### ISSUE TITLE
Is there any way to see var types?

### ISSUE THREAD

**Issue by jstoxrocky** [nwolverson/vscode-ide-purescript#211]
I feel like I'm frequently doing something like this (adding an int to a known non-semiring) just to see what type I'm dealing with.
Is there any way to like, hover over a var name and see the type?

<img width="830" alt="Screen Shot 2022-12-18 at 4 41 32 PM" src="https://user-images.githubusercontent.com/5459666/208318457-4b696253-703e-4ab2-b0f4-8333d28799db.png">

**Comment by nwolverson** [comment]
No, this is not a feature provided by the compiler (`purs ide server`) at this point. Currently you can see types only for (exported) top level identifiers and their usages.

This is something that has been desired for a long time, hopefully it will happen at some point - check compiler repo.

Closing as duplicate of #162
